transform/person_indexer.go

```diff
@@ -34,9 +34,12 @@ func (m *PersonIndexer) Index(resource *models.Person, doc solr.Document) solr.D
 	// 4. institution
 	doc.Set("institution_label_ssim", m.retrieveLabels(resource.InstitutionOrgs))
 
-	// 5. institution
+	// 5. countries
 	doc.Set("countries_label_ssim", m.retrieveLabels(resource.Countries))
 
+	// 6. subtype
+	doc.Set("person_subtype_ssi", m.retrieveLabels(resource.Subtypes))
```

Comment:
If subtypes is multi-valued, should this be `_ssim`?

Reply:
Deferring to your solr expertise.

Reply:
Fixed.

Reply:
OK!